link-qkv2.x

```diff
@@ -0,0 +1,105 @@
+INCLUDE memory.x
+/* Provides weak aliases (cf. PROVIDED) for device specific interrupt handlers */
+/* This will usually be provided by a device crate generated using svd2rust (see `device.x`) */
+INCLUDE device.x
+
+PROVIDE(_stext = ORIGIN(REGION_TEXT));
+PROVIDE(_stack_start = ORIGIN(REGION_STACK) + LENGTH(REGION_STACK));
+PROVIDE(_max_hart_id = 0);
+PROVIDE(_hart_stack_size = 2K);
+PROVIDE(_heap_size = 0);
+
+/* fault handlers */
+PROVIDE(InstructionMisaligned = ExceptionHandler);
+PROVIDE(InstructionFault = ExceptionHandler);
+PROVIDE(IllegalInstruction = ExceptionHandler);
+PROVIDE(Breakpoint = ExceptionHandler);
+PROVIDE(LoadMisaligned = ExceptionHandler);
+PROVIDE(LoadFault = ExceptionHandler);
+PROVIDE(StoreMisaligned = ExceptionHandler);
+PROVIDE(StoreFault = ExceptionHandler);;
+PROVIDE(UserEnvCall = ExceptionHandler);
+PROVIDE(MachineEnvCall = ExceptionHandler);
+
+/* core interrupt handlers */
+PROVIDE(NonMaskableInt = DefaultHandler);
```

Comment:
These are already defined in pac crate.

Reply:
Did you mean the file "link-highcode.x" in qingke-rt crate?
This link file add .trap1kb section to place *(.trap) code.
```
    .trap1kb  : ALIGN(1024)
    {
        *(.trap)
    } >FLASH AT>FLASH
```
QingKe V2 vector table base address needs to be 1KB aligned.